tests/InetSocketTestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace KS;
+
+abstract class InetSocketTestCase extends SocketTestCase

```

Comment:
Not a big deal, but I see this is a base class for the "real" `InetSocketTest`. The convention in PHP is simply to call abstract classes `Abstract`. Then it becomes clear what the relationship is between the abstract and the concrete: `class InetSocketTest extends AbstractInetSocketTest`. In other words, `InetSocketClass` uses `AbstractInetSocketTest` as a foundation to build its tests on.